types/heatmap.js/heatmap.js-tests.ts

```diff
@@ -1,42 +1,283 @@
+// tslint:disable-next-line no-object-literal-type-assertion
+const container = {} as HTMLElement;
```

Comment:
You could also do `declare const container: HTMLElement;`

Reply:
Ah, nice. I was wondering about that. 86ba50fb2fc45c6e956b4a47354f4ac1584a61f2